tests/tests.yaml

```diff
@@ -0,0 +1,14 @@
+charm_name: cinder-three-par
+tests:
+  - tests.tests_cinder_three_par.CinderThreeParTest
+configure:
+  - zaza.openstack.charm_tests.keystone.setup.add_demo_user
+gate_bundles:
```

Comment:
Technically this should be focal-xena, focal-yoga, and jammy-yoga. As we create different openstack release tracks it should look different. E.g. for the xena track/branch it will look like focal-xena, focal-wallaby and for wallaby track/branch it will look like focal-wallaby, focal-victoria, for the victoria track/branch it will look like focal-victoria, focal-ussuri, and finally for the ussuri track/branch it should be focal-ussuri, bionic-ussuri (no train as I believe our initial support was included for Ussuri)